<ui version="4.0" >
 <class>AlarmDialog</class>
 <widget class="QDialog" name="AlarmDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="../icons.qrc" >:/icons/fosdem.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="message" >
         <property name="text" >
          <string>Alarm message goes here</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QPushButton" name="appPB" >
         <property name="text" >
          <string>App</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="snoozePB" >
         <property name="text" >
          <string>Snooze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopPB" >
         <property name="text" >
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections/>
</ui>
